scheduler/core/gocron_scheduler.go

```diff
@@ -271,7 +271,11 @@ func (g *GoCronScheduler) AddFixedIntervalJob(ctx context.Context, job *GoCronJo
 	var jobFunc cron.TimedFuncJob
 	jobFunc = job.Run
 
-	entryID := g.cron.ScheduleTimedJob(cron.ConstantDelaySchedule{Delay: d}, jobFunc)
+	var lastExecTime time.Time
+	if job.lastTime != nil {
```

Comment:
nit: should this field name change to job.lastExecTime too?

Reply:
Changing this would mean maintaining backward compat. I will clean this up when i next make change to this file

Reply:
On second thought, the JobStore will get loaded again after new version, so it will start using the new field after the restart, so we don't have to worry about backward compat.

Reply:
Awesome, can you verify that there are indeed no problems on restart locally?